src/Components/QuickGrid/Microsoft.AspNetCore.Components.QuickGrid/src/Columns/Align.cs

```diff
@@ -9,17 +9,27 @@ namespace Microsoft.AspNetCore.Components.QuickGrid;
 public enum Align
 {
     /// <summary>
-    /// Justifies the content against the start of the container.
+    /// Justifies the content against the start of the container
     /// </summary>
-    Left,
+    Start,
 
     /// <summary>
     /// Justifies the content at the center of the container.
     /// </summary>
     Center,
 
     /// <summary>
-    /// Justifies the content at the end of the container.
+    /// Justifies the content at the end of the container
+    /// </summary>
+    End,
+
+    /// <summary>
+    /// Justifies the content against the left of the container.
+    /// </summary>
+    Left,
+
+    /// <summary>
+    /// Justifies the content at the right of the container.
```

Comment:
Super duper nit: Could we add a '.' at the end of each comment that doesn't have one?